### PR TITLE
Update guide for Linux preview

### PR DESCRIPTION
**Pull request by aiordache**
- updated the deb package URL
- added a few known issues
- removed the feedback section until the preview goes public

### CODE REVIEW

**Comment by p1-0tr** [comment]
LGTM

File: desktop/linux/index.md [hunk 1]
```diff
@@ -23,15 +23,16 @@ To install Docker Desktop for Linux:
...
-
-  ```console
-  $ curl https://desktop-stage.docker.com/linux/main/amd64/73701/docker-desktop.deb --output docker-desktop.deb
-  $ sudo apt install ./docker-desktop.deb
-  ```
+    ```console
+    $ curl https://desktop-stage.docker.com/linux/main/amd64/73772/docker-desktop.deb --output docker-desktop.deb
+    $ sudo apt install ./docker-desktop.deb
+    ```
+3. Check the user belongs to the `docker` and `kvm` groups. The host might need to be restarted for the group configuration to be loaded.
```
Comment (usha-mandya) [inline desktop/linux/index.md:30]:
```suggestion
3. Check whether the user belongs to `docker` and `kvm` groups. You may need to restart the host to load the group configuration.
```

File: desktop/linux/index.md [hunk 2]
```diff
@@ -136,7 +139,12 @@ To remove Docker Desktop for Linux, run:
 $ sudo apt remove docker-desktop
 ```
 
-## Feedback
 
-Your feedback is important to us. Let us know what you think by adding a comment
-on our [Docker for Linux public roadmap](https://github.com/docker/roadmap/issues/39){: target="_blank" rel="noopener" class="_"} issue.
+## Known issues
```
Comment (usha-mandya) [inline desktop/linux/index.md:143]:
```suggestion
## Known issues

```

Comment (usha-mandya) [reply desktop/linux/index.md:143]: nit: add extra line after heading

File: desktop/linux/index.md [hunk 3]
```diff
@@ -136,7 +139,12 @@ To remove Docker Desktop for Linux, run:
...
+ - `Reset to factory defaults` currently does not work
```
Comment (usha-mandya) [inline desktop/linux/index.md:147]:
```suggestion
 - The **Reset to factory defaults** option on the **Troubleshoot** page currently does not work.
```

File: desktop/linux/index.md [hunk 4]
```diff
@@ -136,7 +139,12 @@ To remove Docker Desktop for Linux, run:
...
+ - At the end of installation, apt produces an error due to installing a downloaded package. This should be ignored.
```
Comment (usha-mandya) [inline desktop/linux/index.md:148]:
```suggestion
 - At the end of the installation process, `apt` displays an error due to installing a downloaded package. You can ignore this error message.
```

File: desktop/linux/index.md [hunk 5]
```diff
@@ -136,7 +141,13 @@ To remove Docker Desktop for Linux, run:
...
 
-## Feedback
 
-Your feedback is important to us. Let us know what you think by adding a comment
-on our [Docker for Linux public roadmap](https://github.com/docker/roadmap/issues/39){: target="_blank" rel="noopener" class="_"} issue.
+## Known issues
+
+ - The **Reset to factory defaults** option on the **Troubleshoot** page currently does not work.
+ - At the end of the installation process, `apt` displays an error due to installing a downloaded package. You can ignore this error message.
+  ```
```
Comment (usha-mandya) [inline desktop/linux/index.md:149]:
```suggestion
  
```console
```

Comment (usha-mandya) [reply desktop/linux/index.md:149]: nit: a line before the error message

**Review by usha-mandya — APPROVED** [review]
LGTM. Thank you @aiordache

**Pull request merged**